assets/css/ie-editor.css

```diff
@@ -1606,7 +1606,13 @@ pre.wp-block-verse {
 	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
 	font-weight: 300;
 	font-size: 4rem;
-	line-height: 1.3;
+	line-height: 4rem;
```

Comment:
Is this meant to be rems, instead of unitless?

Reply:
Should be unitless I think